docs/others/fragment.md

```diff
@@ -0,0 +1,72 @@
+```eval_rst
+.. include:: /header.rst 
+:github_url: |github_link_base|/others/fragment.md
+```
+
+# Fragment
+
+This is something like [Fragment on Android](https://developer.android.com/guide/components/fragments)
+or [UiViewController on iOS](https://developer.apple.com/documentation/uikit/uiviewcontroller) - to help developers
+better handles code structure, and object lifecycle.
+
```

Comment:
Some more info wold be great here: e.g. 
- what is a fragment
- it consists of a fragment manager that can create/delete fragments attached to it...

Reply:
I will copy many sentences from Android Developers 😆  the concept is identical. https://developer.android.com/guide/fragments

Reply:
No problem with that. To goal is to have a clear easy to digest description about what this feature is.